public/developerstats.php

```diff
@@ -57,14 +72,13 @@
 
         echo "<tr class='do-not-highlight'>";
         echo "<th></th>";
-        echo "<th><a href='/developerstats.php?t=6&f=$devFilter'>Name</a>" . ($type == 6 ? "*" : "") . "</th>";
-        echo "<th class='text-right whitespace-nowrap'><a href='/developerstats.php?t=3&f=$devFilter'>Open Tickets</a>" . ($type == 3 ? "*" : "") . "</th>";
-        echo "<th class='text-right whitespace-nowrap'><a href='/developerstats.php?f=$devFilter'>Achievements</a>" . ($type == 0 ? "*" : "") . "</th>";
-        echo "<th class='text-right whitespace-nowrap'><a href='/developerstats.php?t=4&f=$devFilter'>Ticket Ratio (%)</a>" . ($type == 4 ? "*" : "") . "</th>";
-        echo "<th class='text-right'><a href='/developerstats.php?t=2&f=$devFilter' title='Achievements unlocked by others'>Yielded Unlocks</a>" . ($type == 2 ? "*" : "") . "</th>";
-        echo "<th class='text-right'><a href='/developerstats.php?t=1&f=$devFilter' title='Points gained by others through achievement unlocks'>Yielded Points</a>" . ($type == 1 ? "*" : "") . "</th>";
-        echo "<th class='text-right'><a href='/developerstats.php?t=7&f=$devFilter' title='Set claims currently active'>Active Claims</a>" . ($type == 7 ? "*" : "") . "</th>";
-        // echo "<th class='text-right whitespace-nowrap'><a href='/developerstats.php?t=5'>Last Login</a>" . ($type == AchievementType::UNOFFICIAL ? "*" : "") . "</th>";
+        echo "<th><a href='/developerstats.php?t=6&f=$devFilter'>Name</a>$orderedByName</th>";
+        echo "<th class='text-right whitespace-nowrap'><a href='/developerstats.php?t=3&f=$devFilter'>Open Tickets</a>$orderedByOpenTickets</th>";
+        echo "<th class='text-right whitespace-nowrap'><a href='/developerstats.php?f=$devFilter'>Achievements</a>$orderedByAchievements</th>";
+        echo "<th class='text-right whitespace-nowrap'><a href='/developerstats.php?t=4&f=$devFilter' title='Ticket Resolved for Others'>Tickets Resolved For Others</a>$orderedByResolvedTickets</th>";
```

Comment:
Wondering if there are any other headings we can use (though keep `title` as the same). This heading (and all its cells) seems to stick out from all the others because of its character count.

Possible alternatives:

Assisted Resolves
Collab Ticket Fixes
External Ticket Fixes
Helped Tickets

Reply:
Yeah I wasn't happy with the heading being so long but couldn't think of a clear short alternative. 
I still want to call it `Ticket Karma` but that used to be a thing previously on the individual stats page and was seen negatively thus removed so I think when I asked a long time ago it was advised against but maybe now it would be okay

Reply:
What about putting a newline/<br> in the middle of it?

Reply:
<img width="762" alt="Screenshot 2023-05-05 at 12 54 21 PM" src="https://user-images.githubusercontent.com/3984985/236519637-27fc7e5c-1c22-47ca-a059-2a0a6c10da51.png">

max-width of 120px and the whitespace-nowrap classname removed.